### ISSUE TITLE
Incorrect evaluation of isNumeric SPARQL function

### ISSUE THREAD

**Issue by jakubklimek** [openlink/virtuoso-opensource#646]
See the [specification](https://www.w3.org/TR/sparql11-query/#func-isNumeric) and try the `xsd:nonNegativeInteger` example:
``` sparql
SELECT ?numeric ?isNumeric {
BIND("12"^^xsd:nonNegativeInteger as ?numeric)
BIND(isNumeric(?numeric) as ?isNumeric)
} 
```
yields
```
numeric	                                                      isNumeric
"12"^^<http://www.w3.org/2001/XMLSchema#nonNegativeInteger>   0
```

1) First of all, it is weird that the result of this function is a `number` and not a `boolean` value
2) But more importantly, it should be `true` (`1`) not `false` (`0`)

**Comment by HughWilliams** [comment]
I have reported to development to look into ...

**Comment by jakubklimek** [comment]
Fixed in d76ab20f8cdee8b7660bf1b653fda85d08e5153c
